pyproject.toml

```
@@ -35,16 +35,16 @@ classifiers = [
 requires-python = ">= 3.11, < 4"
 dependencies = [
   "python-dateutil >= 2.8.2, < 3.0",
-  "typing-extensions == 4.11.0",
+  "typing-extensions => 4.11.0, < 5.0.0",
   # Make sure to update the version for cross-referencing also in the
   # mkdocs.yml file when changing the version here (look for the config key
   # plugins.mkdocstrings.handlers.python.import)
   "frequenz-sdk == v1.0.0-rc6",
```

The version should not use the `v` prefix. While we are at it, upgrade to rc601 to get some bug fixes:
```suggestion
  "frequenz-sdk == 1.0.0-rc601",
```